test/suites/integration/ami_test.go

```diff
@@ -64,8 +64,13 @@ var _ = Describe("AMI", func() {
 	})
 	It("should use the most recent AMI when discovering multiple", func() {
 		// choose an old static image
+		parameters, err := env.SSMAPI.GetParametersByPath(&ssm.GetParametersByPathInput{
```

Comment:
We should also expect a certain length from these responses. Else, we might get a panic when we try to index if the length is too short